Appearance/AppDelegate.swift

```diff
@@ -61,6 +61,17 @@ class AppDelegate: NSObject, NSApplicationDelegate {
         hooksController.start()
 
         hooksController.add { [weak self] theme in
+            os_log("Will update current colorscheme file, colorscheme is %@",
+                   theme.colorScheme.description)
+            let filename = FileManager.default.currentColorschemeFile.absoluteURL
+            do {
+                try theme.colorScheme.description.write(to: filename, atomically: true, encoding: String.Encoding.utf8)
+            } catch {
+                os_log("error: %@", error.localizedDescription)
+                self?.alert(title: "Error updating current colorscheme file: \(filename)", body: error.localizedDescription)
+            }
+            os_log("Did update current colorscheme file")
+
```

Comment:
You could att this as a new call to add above with its own closure like this.

```suggestion
        hooksController.add { [weak self] theme in
            os_log("Will update current colorscheme file, colorscheme is %@",
                   theme.colorScheme.description)
            let filename = FileManager.default.currentColorschemeFile.absoluteURL
            do {
                try theme.colorScheme.description.write(to: filename, atomically: true, encoding: String.Encoding.utf8)
            } catch {
                os_log("error: %@", error.localizedDescription)
                self?.alert(title: "Error updating current colorscheme file: \(filename)", body: error.localizedDescription)
            }
            os_log("Did update current colorscheme file")
        }

        hooksController.add { [weak self] theme in
```

Each call to add will add a new hook to the instance of `HooksController`.